xc/xc7/yosys/conv.tcl

```diff
@@ -1,20 +0,0 @@
-yosys -import
```

Comment:
@olofk The failure is due to the absence of the conv.tcl.
I see you have moved it to the synth.tcl so to have everything computed in one call only. The problem with this is the current handling of dependencies in cmake, as the conv.tcl script cannot be found anymore, the build fails.
[devices.cmake](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/common/cmake/devices.cmake) holds all the cmake functions to create the devices targets, and there are the two separate commands to call the synth.tcl and conv.tcl.


IIRC the original reason to split synth and conv scripts was to apply the `split_inouts` script between the two, as the inouts signals need to be preprocessed to continue with the further stage of the build, @mkurc-ant can confirm that.

Reply:
ok, so thinking about it again, the python script+conv.tcl together basically forms a json -> eblif bridge. I can see the case for having this separate.

We can drop this change and I will take care of it on the edalize side instead

Reply:
I agree, that's the reason for that. Ultimately we would want the split process to be part of the Yosys flow. This could be done via a plugin: https://github.com/SymbiFlow/symbiflow-arch-defs/issues/1172